src/main.rs

```diff
@@ -26,16 +29,15 @@ use vc_util::{
     vc_signing_input, vc_signing_input_hash, AliasTuple, CredentialParams,
 };
 
-use asset_util::{collect_assets, CertifiedAssets};
-use ic_cdk_macros::post_upgrade;
-
 /// We use restricted memory in order to ensure the separation between non-managed config memory (first page)
 /// and the managed memory for potential other data of the canister.
 type Memory = RestrictedMemory<DefaultMemoryImpl>;
 type ConfigCell = StableCell<IssuerConfig, Memory>;
 type EarlyAdoptersMap = StableBTreeMap<Principal, EarlyAdopterData, VirtualMemory<Memory>>;
+type EventsMap = StableBTreeMap<String, EventRecord, VirtualMemory<Memory>>;
```

Comment:
maybe add a description what is the `String`-key in the map?  (or just define a type with a meaningful name yet equal to `String` and use it in the `EventsMap`-definition?)

Reply:
Good point, done!